packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -208,7 +209,7 @@ function persistedProfileMigrationToV11(
     if (!existingProfile?.network) {
         let network: IPersistedNetwork
         const networkId = getNetworkIdFromOldNetworkType(existingProfile?.networkType)
-        if (networkId === NetworkId.Shimmer || networkId === NetworkId.Testnet) {
+        if (networkId === NetworkId.Shimmer || networkId === NetworkId.ShimmerTestnet) {
```

Comment:
In this case, wouldn't it be necessary to check all 3?
`networkId === NetworkId.Shimmer || networkId === 'testnet' || networkId === NetworkId.ShimmerTestnet`

Reply:
`networkId` comes from `getNetworkIdFromOldNetworkType` which doesnt return `testnet` anymore so its shouldnt be necessary